server/src/main/resources/keywhiz-development.yaml.postgres

```diff
@@ -86,6 +86,8 @@ readonlyDatabase:
 migrationsDir:
   db/postgres/migration
 
+statusCacheExpiry: PT60S
```

Comment:
Hard-code this, or else make sure to provide a decent default value. We have too many config options as it is already.